src/System.Reflection.TypeExtensions/src/System/Reflection/TypeExtensions.CoreCLR.cs

```diff
@@ -279,6 +314,18 @@ public static MethodInfo GetBaseDefinition(MethodInfo method)
         }
     }
 
+    public static class ModuleExtensions
+    {
+        public static bool TryGetModuleVersionId(Module module, out Guid mvid)
```

Comment:
Yes, it didn't affect testing because everything goes through the reference assembly. 

I actually left off the 'this' in implementation for things that have an exact instance counterpart in mscorlib (otherwise code looks like infinite recursion,  but that doesn't apply here, and in fact on desktop you reference the impl and need to see this new method as an extension.

Will fix.